backend/uasmartsignage/src/main/java/deti/uas/uasmartsignage/Services/FileService.java

```diff
@@ -54,9 +70,20 @@ public CustomFile getFileOrDirectoryById(Long id) {
         logger.info("Retrieving file with ID: {}", id);
         CustomFile file = fileRepository.findById(id).orElse(null);
 
+        // Add log to InfluxDB
+        String measurement = "BackendLogs";
+        String operationSource = "FileService";
+        String operation = "RetrieveFileById";
+        String description = "Retrieved file with ID: " + id;
+        String bucket = "changeme";
+        if (!logsService.addBackendLog(measurement,2, operationSource, operation, description, bucket)) {
+            logger.error("Failed to add log to InfluxDB");
+        }
```

Comment:
Hard coding all of these strings can be dangerous. Try to find another way to get this information without hard coding it (most of it).

Reply:
More organized right now but the log information needs to be given somehow